Cargo.toml

```diff
@@ -37,3 +36,6 @@ lazy_static = "1"
 invocation = []
 backtrace = ["error-chain/backtrace"]
 default = ["backtrace"]
+
+[package.metadata.docs.rs]
+features = ["invocation"]
```

Comment:
Hm, I expected it to fail because building with invocation feature [requires a valid path to libjvm in LD_LIBRARY_PATH](https://github.com/jni-rs/jni-rs/blob/master/CONTRIBUTING.md#setup-environment), but `cargo doc --features=invocation --open` worked!

Is it a good practice to show the APIs that require a feature to be enabled? The generated docs unfortunately don't label such APIs at all:
![image](https://user-images.githubusercontent.com/7531824/61641750-5b6f1300-aca8-11e9-8514-71a7c6ac151e.png)

Reply:
Although the usual guidelines do not talk much about this mechanism, I usually include here all features that would extend the API, since at least they'll become visible to everyone who might be looking for them in docs.rs. Tests are not executed in this context, nor are dynamically linked objects required, which is a plus when some features require very specific resources.

Reply:
I agree that it is worth including this API, as it is essential for one of the important use-cases — launching the VM from Rust. It is unfortunate that Rustdoc cannot label such methods automatically. In order to avoid confusion, I think we can add a note as the last paragrapth to the documentation of JavaVM#new, InitArgs and InitArgsBuilder:
```
This API requires "invocation" feature to be enabled, 
see ["Launching JVM from Rust"](struct.JavaVM.html#launching-jvm-from-rust).
```

Reply:
I also think that adding such note is a good idea.